CRM/Mailing/Event/BAO/Subscribe.php

```diff
@@ -227,7 +227,7 @@ public function send_confirm_request($email) {
       'From' => "\"{$domainEmailName}\" <{$domainEmailAddress}>",
       'To' => $email,
       'Reply-To' => $confirm,
-      'Return-Path' => "do-not-reply@$emailDomain",
+      'Return-Path' => CRM_Core_BAO_Domain::getNoReplyEmailAddress(),
```

Comment:
@eileenmcnaughton you need to remove
```
$emailDomain = CRM_Core_BAO_MailSettings::defaultDomain();
```
above at line 203

Reply:
I updated the others - but this one is still used